app/schema/mutations/createApplicationMutation.ts

```diff
@@ -4,38 +4,28 @@ import { commitMutation, graphql } from 'react-relay';
 const mutation = graphql`
   mutation createApplicationMutation($input: CreateApplicationInput!) {
     createApplication(input: $input) {
-      application {
-        formData
-        owner
-        referenceNumber
-        status
-      }
+      clientMutationId
     }
   }
 `;
 
-const createApplicationMutation = (applicationData: {
-  formData: string;
-  referenceNumber: number;
-  status: string;
-}) => {
-  const { formData, referenceNumber, status } = applicationData;
+const createApplicationMutation = (owner: string) => {
   const variables = {
     input: {
       application: {
-        formData,
-        referenceNumber: `${referenceNumber}`,
-        status,
+        owner: owner,
       },
     },
   };
-
   commitMutation(relayEnvironment, {
     mutation,
     variables,
     onError: () => {
       return console.log('CreateApplicationMutation failed');
     },
+    onCompleted: (response) => {
+      console.log(response);
```

Comment:
Is there a better practice for the `onCompleted` callback as opposed to a console.log?

Reply:
@BryanGK yes, probably, that is actually left in there from some debugging. I couldn't figure out how to get it to actually return the response for some reason though since it wasn't a requirement for the sprint I moved on. I can clean it up though if you'd like.

Reply:
nah it's good, we can leave it there for now as I'm sure it'll be used for something come launch